src/gfnff/d3com.f90

```diff
@@ -0,0 +1,24 @@
+! This file is part of xtb.
+!
+! Copyright (C) 2019-2020 Stefan Grimme
+!
+! xtb is free software: you can redistribute it and/or modify it under
+! the terms of the GNU Lesser General Public License as published by
+! the Free Software Foundation, either version 3 of the License, or
+! (at your option) any later version.
+!
+! xtb is distributed in the hope that it will be useful,
+! but WITHOUT ANY WARRANTY; without even the implied warranty of
+! MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+! GNU Lesser General Public License for more details.
+!
+! You should have received a copy of the GNU Lesser General Public License
+! along with xtb.  If not, see <https://www.gnu.org/licenses/>.
+
+module gff_d3com
+      use iso_fortran_env, only : wp => real64
+      use xtb_disp_dftd4, only : r2r4 => r4r2, rcov
```

Comment:
We should probably not forward this and just get rid of the module entirely.
`xtb_disp_dftd4` is already forwarding this from another module.

Reply:
It's gone.